core/src/main/java/tachyon/client/RemoteBlockInStream.java

```diff
@@ -40,6 +44,50 @@
 
   /** The block info of the block we are reading */
   private ClientBlockInfo mBlockInfo;
+
+  /**
+   * For each worker that stores the block, we also have a list of storage directories that pages of
+   * the block are in. While reading, we want to consider the workers in order of storage tier, from
+   * lowest to highest, so we build a sorted list of NetAddress, storageDirId pairs.
+   */
+  private static class WorkerInfoPair implements Comparable<WorkerInfoPair> {
+    public WorkerInfoPair(NetAddress mAddress, long mStorageDirId) {
```

Comment:
Would it make sense to name the arguments without the `m` and remove the `this.`?